wgpu-core/src/instance.rs

```diff
@@ -250,6 +250,44 @@ impl<B: GfxBackend> Adapter<B> {
         }
     }
 
+    pub fn get_swap_chain_preferred_format(
```

Comment:
we'd also need a method on the `Global` object that takes an `AdapterId` and `SurfaceId` and returns the same result